tools/debug_tools/src/debug_ui.rs

```diff
@@ -79,8 +40,8 @@ pub fn initialize_infotext(mut commands: Commands, asset_server: Res<AssetServer
 pub fn change_infotext(
     time: Res<Time>,
     diagnostics: Res<Diagnostics>,
-    // key: Res<Input<KeyCode>>, // add this later to toggle the fps display
     mut fpstext_query: Query<&mut Text, With<FpsText>>,
+    bools: Res<DebugInfo>,
```

Comment:
For more consistency :)
```suggestion
    debug_info: Res<DebugInfo>,
```